024-DoYouEvenSynapse/README.md

```diff
@@ -0,0 +1,34 @@
+# What The Hack - Synapse What The Hack
+
+## Introduction
+Contoso Powerlifting Federation (CPF) are in dire need to modernize their data approach. Among the tasks required by your team, you will be challenged to retrieve the source data and upload it into a modern data warehouse repository in a cleansed/modelled format to be able to report and visualize. 
+
+## Learning Objective
+The purpose of this What The Hack is to learn:
+- About some of the features of Azure Synapse
+- The power of data visualization
+- How to add additional data into Azure Synapse
+- How Machine Learning is incorporated 
+
+## Challenges
+- Challenge 0: **[Pre-requisites - Ready, Set, GO!](Student/00-prereqs.md)**
+   - Prepare your workstation to work with Azure
+- Challenge 1:  **[Gathering Open Powerlifting Data](Student/01-data-gathering.md)**
+   - Retrieve data and upload to Azure using a storage service
+- Challenge 2:  **[Load data into a relational database/warehouse](Student/02-load-data.md)**
+   - Loading the data
+- Challenge 3:  **[Visualization](Student/03-visualization.md)**
+   - Visualizing the data
+- Challenge 4:  **[Incremental Loads](Student/04-incrementals.md)**
+   - Load incrementals
+- Challenge 5:  **[Mask Data for Privacy](Student/05-data-masking.md)**
+   - Putting your mask on
+ - Challenge 6: **[Add New Data Source(s)](Student/06-new-data.md)**
+   - Additional data sources
+- Challenge 7:  **[Leverage Machine Learning](Student/07-ml.md)**
+   - The machines know too much!
+
+## Contributors
+- [James Jones](https://github.com/JamJarchitect/)
+- [Art Rask](https://github.com/artraskmsft)
+- [Orrin Edenfield](https://github.com/OrrinEdenfield)
```

Comment:
Please add Frank Lopez to the end here. No need to link it to anything but we should have attribution.

Reply:
Done